opendata_collector_module/metrics_opendata_collector/main.py

```diff
@@ -5,7 +5,7 @@
 #  Copyright (c) 2017-2020 Estonian Information System Authority (RIA)
 #
 #  Permission is hereby granted, free of charge, to any person obtaining a copy
-#  of this software and associated documentation files (the "Software"), to deal
+#  of this software and associated documentation files (the "Software"), to deal # noqa
```

Comment:
🤔 Why noqa is needed here? 😅

Reply:
Just to mute Flake8 error `line too long (80 > 79 characters)` :)